modules/ding_frontend/ding_frontend.strongarm.inc

```diff
@@ -153,17 +153,11 @@ background-attachment: fixed;',
   $strongarm->api_version = 1;
   $strongarm->name = 'language_negotiation_language';
   $strongarm->value = array(
-    'locale-user' => array(
```

Comment:
What is this, that is removed?

Reply:
Here is configuration of language detection (/admin/config/regional/language/configure).